models/iaf_psc_alpha.h

```diff
@@ -88,12 +88,13 @@ relevant measures analytically.
    matrix objects.
 
 .. note::
-   If tau_m is very close to tau_syn_ex or tau_syn_in, the model
-   will numerically behave as if tau_m is equal to tau_syn_ex or
-   tau_syn_in, respectively, to avoid numerical instabilities.
-   For details, please see
-   <https://github.com/nest/nest-simulator/blob/master/doc/model_details/IAF_neurons_singularity.ipynb>`_
-   in the NEST source code (docs/model_details).
+
+   If `tau_m` is very close to `tau_syn_ex` or `tau_syn_in`, the model
+   will numerically behave as if `tau_m` is equal to `tau_syn_ex` or
+   `tau_syn_in`, respectively, to avoid numerical instabilities.
+
+For implementation details see the
+`IAF_neurons_singularity <../model_details/IAF_neurons_singularity.ipynb>`_ notebook.
```

Comment:
This sentence should be part of the note above, because it explains the implementation of the mechanism described in the note.